pkg/controllers/management/nodedriver/dynamic_driver.go

```diff
@@ -176,6 +176,24 @@ func (d *Driver) Exists() bool {
 	return err == nil
 }
 
+func (d *Driver) Excutable() error {
+	if d.name == "" {
+		return fmt.Errorf("Empty driver name")
+	}
+
+	if d.builtin {
+		return nil
+	}
+
+	binaryPath := path.Join(binDir(), d.name)
+	_, err := os.Stat(binaryPath)
+	if err != nil {
+		return fmt.Errorf("Driver %s not found", binaryPath)
+	}
+	err = exec.Command(binaryPath).Start()
+	return errors.Wrapf(err, "Driver binary %s couldn't execute", binaryPath)
```

Comment:
although perhaps errors.wrapf would return nil when err is nil, this construct doesn't look clean in the code. Lets check for error state, and return nil if error is nil, explicitly